.github/workflows/ci.yml

```diff
@@ -75,3 +75,21 @@ jobs:
       run: |
         yum -y install gcc make
         make REDIS_CFLAGS='-Werror'
+
+  schema-validator:
```

Comment:
don't forget to change to "daily" and add `--verbose --fail-commands-not-all-hit` to the validator args

Reply:
also `exit(1)` if any command from COMMAND DOCS is missing a schema (can be done even before processing the reqres files)